onchain/Scarb.lock

```diff
@@ -4,3 +4,17 @@ version = 1
 [[package]]
 name = "joyboy"
 version = "0.1.0"
+dependencies = [
+ "openzeppelin",
+ "snforge_std",
+]
+
+[[package]]
+name = "openzeppelin"
+version = "0.11.0"
+source = "git+https://github.com/OpenZeppelin/cairo-contracts.git?tag=v0.11.0#a83f36b23f1af6e160288962be4a2701c3ecbcda"
+
+[[package]]
+name = "snforge_std"
```

Comment:
snforge should be a dev dependency: https://foundry-rs.github.io/starknet-foundry/getting-started/first-steps.html.